src/test/java/tests/ContextMenuTest.java

```diff
@@ -0,0 +1,29 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ContextMenuTest extends BaseTest {
+
+    /* 1. Правый клик по элементу
+       2. Валидация текста на алерте
+       3. Закрытие алерта */
+
+    private static final String TEXT_ON_ALERT = "You selected a context menu";
+
+    @Test
+    public void checkTextOnAlertTest() {
+        contextMenuPage.openPage();
+        contextMenuPage.clickTheContextMenu();
+        Assert.assertEquals(contextMenuPage.getAlertText(), TEXT_ON_ALERT);
+    }
+
+    @Test
+    public void closeAlertTest() {
+        contextMenuPage.openPage();
+        contextMenuPage.clickTheContextMenu();
+        contextMenuPage.closeAlert();
+        //don't know what kind of assert I could place here
```

Comment:
we can create a method on page which returning true/false if alert is displayed, like that:

`public boolean isAlertPresent() 
{ 
    try 
    { 
        driver.switchTo().alert(); 
        return true; 
    }   // try 
    catch (NoAlertPresentException Ex) 
    { 
        return false; 
    }   // catch 
}   // isAlertPresent()`